src/lib/fields/CreatorsOrContributorsField.js

```diff
@@ -113,30 +117,15 @@ export class ContributorsSerializer extends CreatorsOrContributorsSerializer {
    * @param {object} record - with contributors in frontend format
    * @returns {object} record - with contributors in API format
    */
-  serialize(record, defaultValue) {
-    const recordContributors = _get(record, this.fieldpath, defaultValue);
-    // Remove contributors with only a type
-    // Note: we have to do this because type is filled by default, but
-    // contributors is an optional field
-    let contributors = recordContributors.filter((contributor) => {
-      return !(
-        Object.keys(contributor).length === 1 &&
-        contributor.hasOwnProperty('type')
-      );
-    });
-    contributors = this.serializeCreatorsOrContributors(contributors);
-    // Did we filter out / change contributors?
-    if (!_isEqual(contributors, recordContributors)) {
-      if (contributors.length === 0) {
-        // Yes and now it is empty so we need to strip it
-        delete record.metadata.contributors;
-        return record;
-      } else {
-        // Yes and we simply restructured the identifiers
-        return { ...record, metadata: { ...record.metadata, contributors } };
-      }
-    } else {
-      return record;
-    }
+  serialize(record) {
+    const recordContributors = _get(
+      record,
+      this.fieldpath,
+      this.defaultSerializedValue
+    );
+    let contributors = this.serializeCreatorsOrContributors(recordContributors);
+    return _isEmpty(contributors)
+      ? record
+      : { ...record, metadata: { ...record.metadata, contributors } };
```

Comment:
I deleted, because I actually didn't feel like we needed to put a default value for the `type` as it's not mandatory and it was adding an unnecessary business logic in the serialization time...We can discuss IRL about that :)